site/static/compare/script.js

```diff
@@ -433,6 +387,77 @@ app.component('test-cases-table', {
 </div>
 `
 });
+
+const SummaryPercentValue = {
+    props: {
+        value: Number,
+    },
+    template: `
+<span>{{ value.toFixed(2) }}%</span>
+`
+};
+const SummaryRange = {
+    props: {
+        range: Array,
+    },
+    template: `
+<div v-if="range.length > 0">
+  [<SummaryPercentValue :value="range[0]" />, <SummaryPercentValue :value="range[1]" />]    
+</div>
+<div v-else>-</div>
+`, components: {SummaryPercentValue}
+};
+const SummaryCount = {
+    props: {
+        cases: Number,
+        benchmarks: Number
+    },
+    template: `
+<span :title="cases + ' test case(s), ' + benchmarks + ' unique benchmark(s)'">{{ cases }} ({{ benchmarks }})</span>
+`
+};
+
+app.component('summary-table', {
+    props: ['cases'],
+    template: `
+<table class="summary-table">
+    <thead>
+      <th>Range</th>
+      <th>Mean</th>
+      <th>Count</th>
+      <th><!-- icon --></th>
+    </thead>
+    <tbody>
+        <tr class="positive">
+            <td><SummaryRange :range="cases.regressions.range" /></td>
+            <td><SummaryPercentValue :value="cases.regressions.average" /></td>
+            <td><SummaryCount :cases="cases.regressions.count" :benchmarks="cases.regressions.benchmarks" /></td>
+            <td>
+              <svg style="width:16px;height:16px" viewBox="0 0 24 24">
```

Comment:
Shall we switch to the emojis we use in the markdown tables? That will not only simplify this code but also make it easier to match between the two sources.

Reply:
Yes please! ✅

Reply:
Good idea!